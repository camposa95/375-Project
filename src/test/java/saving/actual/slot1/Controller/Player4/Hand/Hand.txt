GRAIN: 1
BRICK: 0
WOOL: 0
ORE: 0
LUMBER: 2
