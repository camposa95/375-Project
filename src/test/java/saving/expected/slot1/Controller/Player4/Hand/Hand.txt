BRICK: 0
WOOL: 0
LUMBER: 2
ORE: 0
GRAIN: 1
